### PR TITLE
add reference page for supported normalization in authz

### PR DESCRIPTION
**Pull request by yangminzhu**
This adds a new page in the security reference section for all the supported normalization behaviors in the authorization policy.

cc @myidpt @justinpettit 

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[x] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/reference/config/security/normalization/index.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+title: Authorization Policy Normalization
+description: Describes the supported normalizations in authorization policies.
+weight: 40
+owner: istio/wg-security-maintainers
+test: n/a
+---
+
+This page describes all the supported normalizations in the authorization policy. The normalized request will be used for the evaluation of authorization policy.
```
Comment (justinpettit) [inline content/en/docs/reference/config/security/normalization/index.md:9]:
```
```

File: content/en/docs/reference/config/security/normalization/index.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+For more information, refer to the [authorization normalization best practices](/docs/ops/best-practices/security/#customize-your-system-on-path-normalization).
```
Comment (ericvn) [inline content/en/docs/reference/config/security/normalization/index.md:11]: Similar to the comment above remove `the` or add something after the link.

File: content/en/docs/reference/config/security/normalization/index.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+### 3. Multiple forward slashes (`//`, `///`, etc.)
+
+Istio will merge multiple forward slashes to a single forward slash (`/`). For example, the request
+with path `/some//data///abc` will be normalized to `/some/data/abc`.
+
+This is disabled by default but can be enabled with the normalization option `MERGE_SLASHES`.
+
+### 4. Single dot and double dots (`/./`, `/../`)
+
+Istio will resolve single dot `/./` and double dot `/../` according to the [RFC 3986](https://tools.ietf.org/html/rfc3986#section-6).
```
Comment (ericvn) [inline content/en/docs/reference/config/security/normalization/index.md:52]: ~~the~~

File: content/en/docs/reference/config/security/normalization/index.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+The singe dot will be resolved as current directory, and the double dots will be resolved as parent directory.
```
[the remark this answers is not in the document]
Comment (ericvn) [reply content/en/docs/reference/config/security/normalization/index.md:53]: **the** current    **the** parent

File: content/en/docs/reference/config/security/normalization/index.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+This is enabled by default.
+
+## Header name related
+
+This applies to the header name specified in the `request.headers[<header-name>]` condition.
+
+### 1. Case-insensitive matching
+
+Istio authorization policy will compare the header name in case-insensitive approach.
```
Comment (ericvn) [inline content/en/docs/reference/config/security/normalization/index.md:82]: s/in/with a

File: content/en/docs/reference/config/security/normalization/index.md (hunk 1 above)
Comment (yangminzhu) [reply content/en/docs/reference/config/security/normalization/index.md]: thanks for the comment, updated.

File: content/en/docs/ops/best-practices/security/index.md [hunk 6]
```diff
@@ -75,6 +75,8 @@ While these normalization options represent recommendations from HTTP standards
 applications may interpret a URL in any way it chooses to. When using denial policies, ensure that you understand how your application behaves.
 {{< /warning >}}
 
+For a complete list of supported normalizations, please refer to the [authorization policy normalization](/docs/reference/config/security/normalization/).
```
[the remark this answers is not in the document]
Comment (yangminzhu) [reply content/en/docs/ops/best-practices/security/index.md]: thanks for the comment, updated.

**Comment by yangminzhu** [comment]
@justinpettit @ericvn I have addressed the comments, could you take a look again? thanks.

**Review by ericvn — APPROVED** [review]

**Pull request merged**